src/Nethermind/Nethermind.Runner/configs/sepolia.cfg

```diff
@@ -20,5 +20,30 @@
     "PivotNumber": 1301000,
     "PivotHash": "0x0f66dfd68f63261eb91253eacb7df90192c751f85204f60840ddb12ccd91227b",
     "PivotTotalDifficulty": "6921232492055148"
+  },
+  "JsonRpc": {
+    "Enabled": true,
+    "Timeout": 20000,
+    "Host": "127.0.0.1",
+    "Port": 8545,
+    "EnabledModules": [
```

Comment:
this is same as DefaultModules, we can drop it?

Reply:
Actually yes, removed